### DISCUSSION TITLE
How to "refresh" hook values when a parent component changes?

### DISCUSSION THREAD

**Discussion by straightupjac** [wevm/wagmi#229] [Q&A]
I have a setup where a user connects their wallet within one component and in later page interactions, I want to be able to pull that account data from the user. 

Right now the useAccount hook is fired up on page load (before the user connects their wallet) and is not triggered again after the user connects their account in the child components. Is there a way to force a refresh of useAccount?

**Comment by tmm** [comment]
Trying to understand the setup you described. Is it something [like this](https://codesandbox.io/s/priceless-borg-e2fksd?file=/src/index.tsx)?

Auto-refresh on account changes should be working.
